rest-api/src/main/java/com/epam/esm/response/JwtResponse.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.esm.response;
+
+import java.util.List;
+
+public class JwtResponse {
+	private String token;
+//	private String type = "Bearer";
```

Comment:
delete

Reply:
I use this field in response, but this value put in AuthenticationController